### PR TITLE
Update share-box js to allow all imprints to use same code

### PR DESCRIPTION
**Pull request by nickcall**
This update allows the passing of the readcube API endpoint from each environment. As BMC and Nature use different endpoints currently.

It also adds a style for any information that may appear if the request sharedIt link is not available.

### CODE REVIEW

File: packages/global-article/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # History
 
+## 2.0.0 (2019-06-26)
+	* Update share-box.js so readcude api endpoint can be passed in allowing all environemts to use same code
```
Comment (saramorueta) [inline packages/global-article/HISTORY.md]: *environments*

File: packages/global-article/js/share-box.js [hunk 2]
```diff
@@ -3,19 +3,22 @@
  *
  */
 
-window.Component.ShareBox = (function (doc) {
+var ShareBox = function () {
 	'use strict';
+	var doc = document;
```
Comment (saramorueta) [inline packages/global-article/js/share-box.js]: I'd suggest replacing the actual variable for its value in all uses. I think there are only 6 uses :)

File: packages/global-article/js/share-box.js [hunk 3]
```diff
@@ -195,8 +204,12 @@ window.Component.ShareBox = (function (doc) {
 			var doi = (config && config.doi) ? config.doi : null;
```
Comment (saramorueta) [inline packages/global-article/js/share-box.js]: I'd add validation for the `url` as well as it's done with the `doi` and avoid calling to `_addGetShareUrlClickEvent ` if it's null.

File: packages/global-article/HISTORY.md (hunk 1 above)
Comment (nickcall) [reply packages/global-article/HISTORY.md]: done

File: packages/global-article/js/share-box.js (hunk 2 above)
Comment (nickcall) [reply packages/global-article/js/share-box.js]: done

File: packages/global-article/js/share-box.js (hunk 3 above)
Comment (nickcall) [reply packages/global-article/js/share-box.js]: have added the appropriate code

**Pull request merged**